.circleci/config.yml

```diff
@@ -79,7 +79,7 @@ executors:
   reactnativeios:
     <<: *defaults
     macos:
-      xcode: &_XCODE_VERSION "13.3.0"
+      xcode: &_XCODE_VERSION "14.3.0"
```

Comment:
I will not change this as we don't have Ruby backward compatibility here and we are not really installing the right version of ruby.